### ISSUE TITLE
Unvanquished 0.52 last meters roadmap

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1186]
Postponed (😭):

- server list not blocking on network IO + optional flood limiter: https://github.com/DaemonEngine/Daemon/pull/283

Milestones

- [x] complete tutorial hud: https://github.com/Unvanquished/Unvanquished/pull/1182
- [x] complete keybind with tutorial-compatible deconstruction bind: https://github.com/Unvanquished/Unvanquished/pull/1208
- [x] spectator tutorial fails to get key names from its team: https://github.com/Unvanquished/Unvanquished/issues/1187
- [x] the great gameplay improvements by @Gireen
- [x] new lasgun by chris807: https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/3
- [x] updated UI: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/16
- [x] responsive UI: https://github.com/Unvanquished/Unvanquished/pull/1106 https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/6 https://github.com/DaemonEngine/Daemon/pull/161
- [x] borderless window: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/23
- [x] display who's the winner: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/28
- [x] fix presets: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/22
- [x] add a message on main menu screen background or something like that to tell the user he is testing the game in development https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/19
- [x] ask username at startup: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/14
- [x] impressive engine bugfixes, improvements and cleanup
- [x] missing surfaces on old GPU: https://github.com/DaemonEngine/Daemon/issues/376
- [x] do not load textures for disabled features: https://github.com/DaemonEngine/Daemon/pull/358 and https://github.com/DaemonEngine/Daemon/pull/385
- [x] do not compile GLSL shaders for disabled features: https://github.com/DaemonEngine/Daemon/pull/356
- [x] really disable tiled renderer when unused: https://github.com/DaemonEngine/Daemon/pull/381
- [x] fill empty window at startup: https://github.com/DaemonEngine/Daemon/issues/373
- [x] do not scale down lightmap https://github.com/DaemonEngine/Daemon/pull/357
- [x] do not scale down UI elements https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/issues/21 https://github.com/Unvanquished/Unvanquished/pull/1230
- [x] input issue on Alt+Tab / Alt+Enter: https://github.com/DaemonEngine/Daemon/issues/365 https://github.com/DaemonEngine/Daemon/pull/382
- [x] macOS crash: https://github.com/Unvanquished/Unvanquished/issues/1228 https://github.com/DaemonEngine/Daemon/pull/401
- [x] fixed normal maps with vega
- [x] broken vis in vega globe room: https://github.com/UnvanquishedAssets/map-vega_src.dpkdir/issues/1
- [x] tyrant-blocking cardboxes in vega: https://github.com/UnvanquishedAssets/map-vega_src.dpkdir/issues/2
- [x] all other asset fixes
- [x] complete support in netradiant: minimaps, navmeshes etc.
- [x] OpenAL log spam: https://github.com/DaemonEngine/Daemon/issues/445
- [x] bots are broken: https://github.com/Unvanquished/Unvanquished/issues/1374
- [x] Voting/tutorial overlap https://github.com/Unvanquished/Unvanquished/issues/1386
- [x] Prevent evolving at unlimited rate https://github.com/Unvanquished/Unvanquished/issues/1387

Testing

- [x] **do RC builds** of both game and updater for every supported platforms
- [x] **play some games together** with RC packages with a brand new profile, default binds etc.

Publishing

- [x] Things to merge at the very last moment https://github.com/Unvanquished/Unvanquished/pull/1392
- [x] Things to merge at the very last moment https://github.com/DaemonEngine/Daemon/pull/458
- [x] regenerate md5sums
- [x] rebuild updater
- [x] build release unizip
- [x] write English blog post and French article
- [x] publish English blog post, spread the word
- [x] submit French article to publication

ping @Unvanquished

**Comment by illwieckz** [comment]
I added a banner on the website homepage so people who come for the release know it's incoming:
https://unvanquished.net/

**Comment by illwieckz** [comment]
It looks like what is remaining for 0.52 is:

- [ ] OpenAL log spam: https://github.com/DaemonEngine/Daemon/issues/445
- [ ] bots are broken: https://github.com/Unvanquished/Unvanquished/issues/1374

**Comment by slipher** [comment]
Two more easy ones:
- [ ] Voting/tutorial overlap https://github.com/Unvanquished/Unvanquished/issues/1386
- [ ] Prevent evolving at unlimited rate https://github.com/Unvanquished/Unvanquished/issues/1387

**Comment by illwieckz** [comment]
- [ ] Things to merge at the very last moment https://github.com/Unvanquished/Unvanquished/pull/1392

**Comment by illwieckz** [comment]
Current status is:

Everything that was wanted required seems to be now merged in `0.52.0/sync`.
We did tested the RC 3 on a server with real players.

I just built the RC 5. I'll upload it later in the day.

Unless we find some regression, it is expected the RC 5 becomes the final release, maybe some bikeshedding commit may happen, like typo fixing or thing like that, or maybe some of those PVS fixes that just got posted, but not that much.

We can now fully focus on producing our communication material for the release and stuff like that.
